pom.xml

```diff
@@ -4,6 +4,7 @@
         <groupId>org.mule.services</groupId>
         <artifactId>mule-core-services-parent</artifactId>
         <version>1.2.0-SNAPSHOT</version>
+        <relativePath></relativePath>
```

Comment:
???

Reply:
piggyback. avoids a warning thrown by maven (https://stackoverflow.com/questions/6003831/parent-relativepath-points-at-my-com-mycompanymyproject-instead-of-org-apache)